deeppavlov/skills/dsl_skill/faq/faq_reader.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import Dict
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.data.dataset_reader import DatasetReader
+
+
+@register('faq_dict_reader')
+class FaqDatasetReader(DatasetReader):
```

Comment:
Could you clarify necessity of this class when FaqDatasetReader exists here:
https://github.com/deepmipt/DeepPavlov/blob/master/deeppavlov/dataset_readers/faq_reader.py
?

I suppose it is better to extend functionality of the deeppvalov's class to avoid duplication of responsibilies and class name clashing.